app/browser/api/ledger.js

```diff
@@ -1162,6 +1162,7 @@ const recoverKeys = (state, useRecoveryKeyFile, key) => {
     return state
   }
 
+  state = aboutPreferencesState.setPreferencesProp(state, 'recoveryBalanceRecalculated', false)
```

Comment:
Could you maybe refactor updates to `recoveryBalanceRecalculated` as its own function inside `aboutPreferencesState`, similar to the other functions dealing with recovery props? Just to keep things consistent :)

Reply:
You got it